containers/ecr-viewer/src/app/layout.tsx

```diff
@@ -28,14 +31,18 @@ export default function RootLayout({
       <html lang="en">
         <head />
         <body className="overflow-x-auto">
-          <div className="position-relative isolate">
-            <AuthSessionProvider>
+          <AuthSessionProvider>
+            <Header>
+              <NavLinks />
+            </Header>
+            <div className="position-relative isolate">
               <ToastProvider>
```

Comment:
the toast provider should also start above the header, it should be available everywhere in the "app"